issue installing indigo

Setting up a new mac with homebrew-cheminformatics 

When I brew install indigo I get the following error:

==> Installing indigo from mcs07/cheminformatics
==> Installing dependencies for mcs07/cheminformatics/indigo: cmake
==> Installing mcs07/cheminformatics/indigo dependency: cmake
==> Downloading https://homebrew.bintray.com/bottles/cmake-3.7.1.el_capitan.bott
######################################################################## 100.0%
==> Pouring cmake-3.7.1.el_capitan.bottle.tar.gz
==> Caveats
Emacs Lisp files have been installed to:
  /usr/local/share/emacs/site-lisp/cmake
==> Summary
🍺  /usr/local/Cellar/cmake/3.7.1: 2,143 files, 29M
==> Installing mcs07/cheminformatics/indigo 
==> Downloading https://github.com/epam/Indigo/archive/indigo-1.2.1.tar.gz
==> Downloading from https://codeload.github.com/epam/Indigo/tar.gz/indigo-1.2.1
######################################################################## 100.0%
==> Downloading https://gist.githubusercontent.com/mcs07/b9b2ccfdb9bc23f60044/ra
######################################################################## 100.0%
==> Downloading https://gist.githubusercontent.com/mcs07/541bcda4cfd6a7d8668b/ra
######################################################################## 100.0%
==> Patching
==> Applying indigo-el-capitan.diff
patching file common/cmake/GetSystemVersion.cmake
patch unexpectedly ends in middle of line
Hunk #1 succeeded at 27 with fuzz 1.
==> Applying indigo-depict-1.2.1.diff
patching file utils/indigo-depict/CMakeLists.txt
==> cmake ../build_scripts/indigo-all
==> make
Last 15 lines from /Users/ebucholtz/Library/Logs/Homebrew/indigo/02.make:
Scanning dependencies of target indigo-inchi-c-test-shared
[ 99%] Building C object indigo-inchi/indigo-inchi/CMakeFiles/indigo-inchi-c-test-shared.dir/tests/c/indigo-inchi-test.c.o
[ 99%] Building C object indigo/api/CMakeFiles/indigo-c-test-shared.dir/tests/c/indigo-test.c.o
[ 99%] Linking CXX executable ../../dist/Mac/10.11/lib/indigo-inchi-c-test-shared
[ 99%] Linking CXX executable ../../dist/Mac/10.11/lib/indigo-c-test-shared
[ 99%] Built target indigo-renderer-shared
Scanning dependencies of target indigo-renderer-c-test-shared
make[2]: *** No rule to make target `/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX10.11.sdk/System/Library/Frameworks/ApplicationServices.framework', needed by `dist/Mac/10.11/lib/indigo-renderer-c-test-shared'.  Stop.
make[2]: *** Waiting for unfinished jobs....
[ 99%] Building C object indigo-renderer/renderer/CMakeFiles/indigo-renderer-c-test-shared.dir/tests/c/indigo-renderer-test.c.o
[ 99%] Built target indigo-c-test-shared
make[1]: *** [indigo-renderer/renderer/CMakeFiles/indigo-renderer-c-test-shared.dir/all] Error 2
make[1]: *** Waiting for unfinished jobs....
[ 99%] Built target indigo-inchi-c-test-shared
make: *** [all] Error 2

--- a/Formula/indigo.rb
+++ b/Formula/indigo.rb
@@ -4,31 +4,14 @@ class Indigo < Formula
 
   homepage 'http://lifescience.opensource.epam.com/indigo/'
 
-  # Add OS X El capitan 10.11 to possible versions
-  patch do
-    url 'https://gist.githubusercontent.com/mcs07/b9b2ccfdb9bc23f60044/raw/632c87764b32b8dad5ff36b9b8d7a77fc7fc0e06/indigo-el-capitan.diff'
-    sha256 '0d7b204819c99b0775a91a529370edfae5d1ed3fc16e37dcb1785db58012005d'
-  end
-
   stable do
-    url 'https://github.com/epam/Indigo/archive/indigo-1.2.1.tar.gz'
-    sha256 'ef28ab89794cc7df44c1fbdff1c7322db71f0feaae2bfe107be9c93dda3b822b'
-
-    # Fix indigo-depict link flags
-    patch do
-      url 'https://gist.githubusercontent.com/mcs07/541bcda4cfd6a7d8668b/raw/c8c4e7e7c68cb2a8f97d49352b565503ae024a81/indigo-depict-1.2.1.diff'
-      sha256 '36ede86428a52c29310ce8a3ac83456575580d7849c6b86e4141e7e6e0ca22c4'
-    end
+    url 'https://github.com/epam/Indigo/archive/indigo-1.3.0beta.tar.gz'
+    version '1.3.0-beta'
+    sha256 'bf5bd7e12cbb904310012e24dffeca43f54d9ebcb0a48ba50987859841eb1d29'
   end
 
   head do
     url 'https://github.com/epam/Indigo.git'
-  end
-
-  devel do
-    url 'https://github.com/epam/Indigo/archive/indigo-1.2.2beta-r37.tar.gz'
-    version '1.2.2beta-r37'
-    sha256 '9f2e1126ce20e003122013c0506154597e7218c5b2ccf44070d555c2b70dfa70'
   end
 
   option 'with-java',   'Build with Java language bindings'
@@ -69,21 +52,20 @@ class Indigo < Formula
     end
 
     if build.with?('java')
-      ver = /SET\(INDIGO_VERSION "(.+?)"/.match(File.read('api/indigo-version.cmake'))[1]
       cd 'api/java' do
-        system "mvn", "versions:set", "-DnewVersion=#{ver}"
+        system "mvn", "versions:set", "-DnewVersion=1.3.0-beta"
         system "mvn", "clean", "package", "install", "-Dmaven.test.skip=true"
-        libexec.install "target/indigo-#{ver}.jar"
+        libexec.install "target/indigo-1.3.0-beta.jar"
       end
       cd 'api/plugins/renderer/java' do
-        system "mvn", "versions:set", "-DnewVersion=#{ver}"
+        system "mvn", "versions:set", "-DnewVersion=1.3.0-beta"
         system "mvn", "clean", "package", "-Dmaven.test.skip=true"
-        libexec.install "target/indigo-renderer-#{ver}.jar"
+        libexec.install "target/indigo-renderer-1.3.0-beta.jar"
       end
       cd 'api/plugins/inchi/java' do
-        system "mvn", "versions:set", "-DnewVersion=#{ver}"
+        system "mvn", "versions:set", "-DnewVersion=1.3.0-beta"
         system "mvn", "clean", "package", "-Dmaven.test.skip=true"
-        libexec.install "target/indigo-inchi-#{ver}.jar"
+        libexec.install "target/indigo-inchi-1.3.0-beta.jar"
       end
       # TODO: Some javadoc bingo bug needs fixing
       # cd 'api/plugins/bingo/java' do

